../../components/omnibox/browser/on_device_tail_model_service.cc:33:23: error: constexpr variable 'kTestPrefix' must be initialized by a constant expression
constexpr std::string kTestPrefix = "face";
                      ^             ~~~~~~
/usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:356:10: note: assignment to member '_M_local_buf' of union with no active member is not allowed in a constant expression
            __c = _CharT();
                ^
/usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.tcc:229:4: note: in call to '&kTestPrefix->_M_use_local_data()'
          _M_use_local_data();
          ^
/usr/bin/../lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/bits/basic_string.h:642:2: note: in call to '&kTestPrefix->_M_construct(&"face"[0], &"face"[4], {{}})'
        _M_construct(__s, __end, forward_iterator_tag());
        ^
../../components/omnibox/browser/on_device_tail_model_service.cc:33:37: note: in call to 'basic_string(&"face"[0], std::allocator<char>())'
constexpr std::string kTestPrefix = "face";
                                    ^
../../components/omnibox/browser/on_device_tail_model_service.cc:33:23: warning: declaration requires an exit-time destructor [-Wexit-time-destructors]
constexpr std::string kTestPrefix = "face";
                      ^
1 warning and 1 error generated.


--- a/components/omnibox/browser/on_device_tail_model_service.cc
+++ b/components/omnibox/browser/on_device_tail_model_service.cc
@@ -30,7 +30,7 @@ namespace {
 // The maximum idle time before the model executor is unloaded from memory.
 constexpr base::TimeDelta kMaxExecutorIdleSeconds = base::Seconds(60);
 // Constants for TFlite model validation.
-constexpr std::string kTestPrefix = "face";
+const std::string kTestPrefix = "face";
 constexpr std::string_view kModelValidationSwitchName =
     "omnibox-on-device-tail-model-validation";
 
